### PR TITLE
BTAT-7409 Updating accessibility for confirm client page

### PR DESCRIPTION
**Pull request by paulfirmstone**
Please review and merge the AT PR https://github.com/hmrc/change-vat-details-acceptance-tests/pull/88 alongside this one.

### CODE REVIEW

File: app/views/agent/confirmClientVrn.scala.html [hunk 1]
```diff
@@ -27,11 +27,13 @@ <h1 class="heading-large">
         @messages("confirmClientVrn.heading")
     </h1>
 
-    <h2 class="bold-medium">@messages("confirmClientVrn.name")</h2>
-    <p>@customerDetails.clientName</p>
+    <dl>
+            <dt class="bold-medium bottom-padding">@messages("confirmClientVrn.name")</dt>
```
Comment (sandypadman) [inline app/views/agent/confirmClientVrn.scala.html]: Tag < dt > is aligned too far away from < dl >

File: app/views/govuk_wrapper.scala.html [hunk 2]
```diff
@@ -53,7 +53,7 @@
 @linksElement = {
 <link rel="stylesheet" type="text/css" href='@routes.Assets.at("stylesheets/palette.css")'>
 <link rel="stylesheet" type="text/css" href='@routes.Assets.at("stylesheets/focus.css")'>
-<link rel="stylesheet" type="text/css" href='@routes.Assets.at("stylesheets/vat-v1.0.css")'>
+<link rel="stylesheet" type="text/css" href='@routes.Assets.at("stylesheets/vat.css?v=0.1")'>
```
Comment (sandypadman) [inline app/views/govuk_wrapper.scala.html]: Not sure if we needed the ?v=0.1 .

File: app/views/agent/confirmClientVrn.scala.html (hunk 1 above)
Comment (paulfirmstone) [reply app/views/agent/confirmClientVrn.scala.html]: I have re-aligned as requested.

File: app/views/govuk_wrapper.scala.html (hunk 2 above)
Comment (paulfirmstone) [reply app/views/govuk_wrapper.scala.html]: This allows to refresh the cache by changing a value instead of renaming the file.

**Pull request merged**